sycl/test/devicelib/cmath_test.cpp

```diff
@@ -0,0 +1,85 @@
+// UNSUPPORTED: windows
```

Comment:
Could you clarify what is the problem with the Windows support?

Reply:
I found device lib infrastructure have problems supporting multiple fallback device lib.
Not matter how many fallback spv files being loaded and linked with users's device code by sycl runtime, only the first one can work on Windows. I have reported this issue to Andrew.
Another block is MSVC complex implementation depends some MSVC internal math functions, we haven't supported them in device lib.

Reply:
> I found device lib infrastructure have problems supporting multiple fallback device lib.
> Not matter how many fallback spv files being loaded and linked with users's device code by sycl runtime, only the first one can work on Windows. I have reported this issue to Andrew.

I think it was an early revision of the patch, and a few things changed since then. I wonder if this is still a problem.

Reply:
Hi, Andrew. I tried the latest intel/llvm on Window and the problem still exists.

Reply:
Please, add a TODO comment with clarification of the problem.

Reply:
OK. I will also file 2 issues for: 1. supporting multiple devicelib on Windows for sycl runtime devicelib infrastructure; 2. provide MSVC internal functions required by complex in device lib;
I am working on MSVC internal functions now.

Reply:
Added "TODO" for Windows support.